examples/06-verif-manual/vm-008-parametric_calculation.py

```diff
@@ -0,0 +1,216 @@
+r"""
+.. _ref_vm8_example:
+
+Parametric Calculation
+----------------------
+Problem Description:
+ - Write a user file macro to calculate the distance d between either nodes
+   or keypoints in ``PREP7``. Define abbreviations for calling the macro and
+   verify the parametric expressions by using the macro to calculate
+   the distance between nodes :math:`N_1` and :math:`N_2` and
+   between keypoints :math:`K_3` and :math:`K_4`.
+
+Reference:
+ - None.
+
+Analysis Type(s):
+ - Parametric Arithmetic.
+
+Element Type:
+ - None.
+
+Geometric Properties(Coordinates):
+ - :math:`N_{\mathrm{1(x,y,z)}} = 1.5, 2.5, 3.5`
+ - :math:`N_{\mathrm{2(x,y,z)}} = -3.7, 4.6, -3`
+ - :math:`K_{\mathrm{3(x,y,z)}} = 100, 0, 30`
+ - :math:`K_{\mathrm{4(x,y,z)}} = -200,25,80`
+
+.. image:: ../../_static/vm8_setup.png
+   :width: 300
+   :alt: VM8 Problem Sketch
+
+Analysis Assumptions and Modeling Notes:
+ - Instead of ``*create``, ``*use``, etc., we have created a class
+   ``Create`` with methods that correspond to each type of simulation.
+   In normal use, this class would most likely already exist, so
+   it is possible to change coordinates and reuse it. The simulation
+   can be checked not just by target values, but also with the simple
+   distances' formula between keypoints as:
+
+   * Calculate distance between two keypoints in the Cartesian coordinate system:
+        :math:`D = \sqrt[2]{(x_2 - x_1)^2 + (y_2 - y_1)^2 + (z_2 - z_1)^2}`
+   * Python representation of the distance formula:
+        .. doctest::
+
+            import math
+            # Define coordinates for keypoints K3 and K4.
+            x1, y1, z1 = 100, 0, 30
+            x2, y2, z2 = -200, 25, 80
+            dist_kp = math.sqrt((x2 - x1)**2 + (y2 - y1)**2 + (z2 - z1)**2)
+            print(dist_kp)
+
+"""
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+# Start MAPDL and import Numpy and Pandas libraries.
+
+# sphinx_gallery_thumbnail_path = '_static/vm8_setup.png'
+
+import numpy as np
+import pandas as pd
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start MAPDL.
+mapdl = launch_mapdl()
+
+
+###############################################################################
+# Pre-Processing
+# ~~~~~~~~~~~~~~
+# Enter verification example mode and the pre-processing routine.
+
+mapdl.clear()
+mapdl.verify()
+_ = mapdl.prep7()
```

Comment:
You can now use ``Mute`` to silence the output:
```suggestion
mapdl.prep7(mute=True)
```

Reply:
I use it `_ = mapdl.prep7()` as it used in many cases of the docs examples, what I have  seen. If it is possible, I woould like to leave it.

Reply:
OK then